frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -347,6 +402,22 @@ export function parseColumnsSpec(colSpec: string): ColumnKey[] {
   }
 }
 
+export function parseColumnOptions(columnOptions: string): ColumnOptionKey[] {
```

Comment:
Add a unit test for parseColumnOptions for this to the existing cells test [file](https://github.com/GoogleChrome/webstatus.dev/blob/main/frontend/src/static/js/components/test/webstatus-overview-cells.test.ts).

You can look at the existing parseColumnsSpec test for inspiration

https://github.com/GoogleChrome/webstatus.dev/blob/88d31eccb7c2bb8ff69fc5cbf70d6706101407ec/frontend/src/static/js/components/test/webstatus-overview-cells.test.ts#L27-L37